genesyscloud/team/resource_genesyscloud_team.go

```diff
@@ -91,53 +97,93 @@ func readTeam(ctx context.Context, d *schema.ResourceData, meta interface{}) dia
 func updateTeam(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	sdkConfig := meta.(*gcloud.ProviderMeta).ClientConfig
 	proxy := getTeamProxy(sdkConfig)
-
 	team := getTeamFromResourceData(d)
-
-	log.Printf("Updating team %s", *team.Name)
+	log.Printf("updating team %s", *team.Name)
 	teamObj, err := proxy.updateTeam(ctx, d.Id(), &team)
 	if err != nil {
-		return diag.Errorf("Failed to update team: %s", err)
+		return diag.Errorf("failed to update team %s : %s", d.Id(), err)
+	}
+	members, ok := d.GetOk("member_ids")
+
+	if ok {
+		memberList := members.([]interface{})
```

Comment:
type assertion may throw in a panic if the optional "member_ids" attribute is not added in resource declaration. ?

Reply:
we check for member_ids is set in the above line, if set (ok) we convert